source/CommonJobs/CommonJobs.Domain.EvalForm/CalificationItem.cs

```diff
@@ -9,7 +9,7 @@ public class CalificationItem
     {
         public string Key { get; set; }
 
-        public decimal Value { get; set; }
+        public decimal? Value { get; set; }
```

Comment:
I checked all the places I could find it. It works fine. We were already considering null or empty values